lightning/config/config.go

```diff
@@ -16,6 +16,7 @@ package config
 import (
 	"encoding/json"
 	"fmt"
+	"go.uber.org/zap"
```

Comment:
code format , we usually import external dependencies separately

Reply:
GoLand helps me 😭